terraso_backend/apps/project_management/graphql/projects.py

```diff
@@ -326,18 +336,16 @@ def mutate_and_get_payload(cls, root, info, project_id, user_id, role):
         project = cls.get_or_throw(Project, "project_id", project_id)
         user = cls.get_or_throw(User, "user_id", user_id)
         request_user = info.context.user
+
+        if not check_project_permission(request_user, "add_member", Context(project=project)):
+            cls.not_allowed_create(
+                model=Membership, msg="User cannot add membership to this project"
```

Comment:
issue: this message is confusing. Do you mean they can't become a member of the project? Or they can't add someone else as a member?

Reply:
The latter. I can update the message to reflect that.